Source/InputMethodController.swift

```diff
@@ -121,14 +121,13 @@ class McBopomofoInputMethodController: IMKInputController {
 
         keyHandler.clear()
         keyHandler.syncWithPreferences()
-        self.handle(state: .Empty(), client: client)
```

Comment:
When activateServer gets called, there's no need to enter the Empty state handler—after this PR, the state machine is guranteed to be in the Empty state (previously it could be in the Deactivated state).

Plus, log analysis showed that some apps may not be able to handle `setMarkedText` remote calls consistently, and the Empty state handler does need to call that.